src/Attestation/Attestation/Commands/GetAzureAttestation.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.Azure.Commands.Attestation.Models;
+using Microsoft.Azure.Commands.Attestation.Properties;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using System;
+using System.Collections;
+using System.Management.Automation;
+using System.Runtime.InteropServices.ComTypes;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Rest.Azure;
+
+namespace Microsoft.Azure.Commands.Attestation
+{
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "Attestation", SupportsShouldProcess = true, DefaultParameterSetName = NameParameterSet)]
```

Comment:
@shleiAmy you can remove `SupportsShouldProcess` from this cmdlet since the cmdlet isn't making any changes on the server

Reply:
remove SupportsShouldProcess